src/components/Products/ProductsItem/ProductsItem.tsx

```diff
@@ -0,0 +1,111 @@
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import { IProduct } from '@types';
+import { ButtonSizes, ButtonVariants, IconsTypes } from '@constants';
+import { Accordion, CustomButton, Icon, Rating, Image } from '@components';
+
+import style from './ProductsItem.module.scss';
+
+interface ProductsItemProps {
+  product: IProduct;
+}
+
+const FREE_SHIPPING = 'Free Shipping';
+
+export const ProductsItem: FC<ProductsItemProps> = ({ product }) => {
+  const {
+    imgs,
+    stock,
+    price,
+    brands,
+    rating,
+    delivery,
+    productId,
+    description,
+    productTitle,
+    originCountry,
+  } = product;
+
+  const productPrice = price.discount ? price.discountedAmount : price.amount;
+
+  const deliveryCost = delivery.cost
+    ? `${delivery.cost} ${price.currency}`
+    : FREE_SHIPPING;
+
+  const deliveryTime = `Delivery in ${delivery.timeframe} day${
+    delivery.timeframe > 1 ? 's' : ''
+  }`;
+
+  return (
+    <article className={style.container}>
+      <Image
+        src={imgs[0]}
+        alt={productTitle}
+      />
+
+      <div className={style.content}>
+        <div className={style.info}>
+          <div className={style.description}>
+            <h2>{productTitle}</h2>
+            <p>{description.short}</p>
+            <Rating rating={rating} />
+          </div>
+          <Accordion>
+            <div className={style.additional_info_container}>
+              <ul className={style.additional_info}>
+                <li>Origin</li>
+                <li>Brand</li>
+                <li>Delivery</li>
+                <li>Stock</li>
+              </ul>
+              <ul className={style.additional_info}>
+                <li>{originCountry}</li>
+                <li>{brands.join(', ')}</li>
+                <li>{delivery.area.join(', ')}</li>
+                <li>{`${stock.amount} ${stock.measure}`}</li>
+              </ul>
```

Comment:
It isn't a good idea to have 2 lists
You always need to be careful with the order of values
+ Think it's better to create an object and go through it and render this list as one

Reply:
Joined 2 lists into 1